src/components/features/decoupled-components/reference-client/reference-client.tsx

```diff
@@ -0,0 +1,70 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Maybe there is a way to fix the error instead of suppressing it with eslint-disable?